### PR TITLE
Added answers for question no 21 and 25

### PR DESCRIPTION
**Pull request by darshant31**
Added answer for 
21. ### How to deep clone a object in javascript ?
25. ### How to start a new Angular project from cli ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,15 +93,25 @@ Once the page is loaded, the browser sends further async requests as needed.
 20. ### What is a Higher Order Component ? Give example and explain the use cases where it should be used.
      Answer To be added
 21. ### How to deep clone a object in javascript ?
-     Answer To be added
+     Assuming that you only have variables and not have any functions in your object,just use:
```
Comment (hjaintech) [inline README.md]: @darshant31 Request you to add a more comprehensive answer that would be required in a real interview. Of course, you could start with this answer, however, the interviewer would expect person to provide a solution that works in all cases without these assumptions.

Comment (darshant31) [reply README.md]: sure

Comment (darshant31) [reply README.md]: done,please have a look at it

File: README.md [hunk 2]
```diff
@@ -93,15 +93,56 @@ Once the page is loaded, the browser sends further async requests as needed.
 20. ### What is a Higher Order Component ? Give example and explain the use cases where it should be used.
      Answer To be added
 21. ### How to deep clone a object in javascript ?
-     Answer To be added
+     	(1)Assuming that you only have variables and not have any functions in your object,just use:
+		var NewObject = JSON.parse(JSON.stringify(OldObject));	
+	(2)Copying an object with the Object.assign() method
```
Comment (hjaintech) [inline README.md:98]: Solution 2 and 3 are incorrect. Both Object.assign() and spread operator would do `shallow copy` and not `deep copy`

**Review by hjaintech — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (darshant31) [reply README.md:98]: Yes, i have attached all the copy method.

**Pull request closed without merging**